package.json

```diff
@@ -50,6 +50,7 @@
     "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
     "@babel/register": "^7.5.5",
+    "@babel/runtime": "^7.5.5",
```

Comment:
ideally the dev dep version is an exact match of the peer dep one

Reply:
@ljharb The issue is that `eslint-plugin-jsx-a11y@6.2.3` installs 7.5.5. 🤔 Should we require the peer dep to be 7.5.5 or 7.0.0?

Reply:
Sure, but ^7 will include that too. I think ^7 is the most widely compatible.